src/Table/TableReflectionInterface.php

```diff
@@ -35,4 +35,6 @@ public function isTemporary(): bool;
     public function getDependentViews(): array;
 
     public function getTableDefinition(): TableDefinitionInterface;
+
+    public function tableExists(): bool;
```

Comment:
nebylo lepší enom `exists` to table je docela redundantní :thinking:, nad tím je sice getTableDefinition, ale ta vracá TableDefinition.
Takže bych řekl `isTableExists` nebo enom `exists`